qwandaq/src/main/java/life/genny/qwandaq/serialization/attribute/Attribute.java

```diff
@@ -1,25 +1,22 @@
 package life.genny.qwandaq.serialization.attribute;
 
-import java.time.LocalDateTime;
-import java.util.List;
+import life.genny.qwandaq.CoreEntityPersistable;
 import life.genny.qwandaq.EEntityStatus;
 import life.genny.qwandaq.converter.ValidationListConverter;
 import life.genny.qwandaq.datatype.DataType;
-import life.genny.qwandaq.CoreEntityPersistable;
 import life.genny.qwandaq.serialization.CoreEntitySerializable;
 import life.genny.qwandaq.validation.Validation;
 
+import java.time.LocalDateTime;
+import java.util.Set;
+
 /*
  * A representation of BaseEntity in the cache
  * 
  * @author Varun Shastry
  */
 public class Attribute implements CoreEntitySerializable {
 	
-	private String dtype;
-	
-	private Long id;
```

Comment:
Not ready to remove Id yet (no fyodor modifications)

Reply:
Fixed